src/pages/CamerasPage/CameraTitle/CameraTitle.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from "./CameraTitle.module.scss";
+import { HTMLAttributes } from "react";
+type Props = {
+    title: string;
+} & HTMLAttributes<HTMLDivElement>;
+
+export const CameraTitle = ({ title, ...props }: Props) => {
+    const className = `${styles.title} ${props.className ?? ""}`;
+
+    return (
+        <div
```

Comment:
this is going to give problems. You have duplicated two times the className prop: one with your custom className and other with the props.className, to prevent it you can do something like this

```javascript
export const CameraTitle = ({ title, className?, ...props }: Props) => {
    const customClassName = `${styles.title} ${className ?? ""}`;

    return (
        <div className={customClassName} {...props}>
            <div className={styles.name}>{title}</div>
            <div className={styles.dot} />
        </div>
    );
};
```

Reply:
I don't understand why, I'm already taking into account that props.className might be undefined.